client/src/Pages/AddReviewPage/AddReviewPage.tsx

```diff
@@ -88,37 +94,45 @@ export const AddReviewPage = () => {
   }, [id]);
 
   return (
-    <Flex>
-      <Title>
-        <BigLabel title={bookData?.title ?? ' '}></BigLabel>
-      </Title>
-      <Description>
-        <Information>
-          <Cover>
-            <BookImage src={bookData?.cover} />
-          </Cover>
-          <Info>
-            <InfoLine>{`${bookData?.name ?? ''} ${bookData?.surname ?? ''}`}</InfoLine>
-            <InfoLine>Data wydania: {format(bookData?.release_date ?? new Date(), 'YYYY-MM-DD')}</InfoLine>
-            <InfoLine>Liczba stron: {bookData?.num_pages}</InfoLine>
-            <InfoLine>ISBN: {bookData?.isbn}</InfoLine>
-            <InfoLine>Srednia ocen: {bookData?.average ?? 0}</InfoLine>
-          </Info>
-        </Information>
-        <Rating>
-          <Text>Twoja ocena:</Text>
-          <StarRating rate={0} getNumberOfStars={setUserStars} />
-        </Rating>
-      </Description>
-      <Review>
-        <ReviewInput
-          placeholder="Napisz co myślisz o tej książce...."
-          onChange={(e) => setUserReview(e.target.value)}
-        ></ReviewInput>
-      </Review>
-      <AddReview>
-        <Button children="Opublikuj" onClick={handleClick}></Button>
-      </AddReview>
-    </Flex>
+    <MyBooksPageWrapper>
+      <Header isLogged={true} />
+      <Navbar />
+      <SearchingBarComponent />
+      <BookPageContainer>
+        <Flex>
+          <Title>
+            <BigLabel title={bookData?.title ?? ' '}></BigLabel>
+          </Title>
+          <Description>
+            <Information>
+              <Cover>
+                <BookImage src={bookData?.cover} />
+              </Cover>
+              <Info>
+                <InfoLine>{`${bookData?.name ?? ''} ${bookData?.surname ?? ''}`}</InfoLine>
+                <InfoLine>Data wydania: {format(bookData?.release_date ?? new Date(), 'YYYY-MM-DD')}</InfoLine>
```

Comment:
Czy to na pewno będzie działąć? Ostatnio taki zapis sie wysypał po deployu, jak chcemy to zostawić to koniecznie po zmergowaniu trzeba bedzie zobaczyc czy to zadzaiała na heroku, bo chyba niestety nie

Reply:
gotowe